docs/user/README.md

```diff
@@ -1,60 +1,60 @@
 
 ## Features
 
-To support telemetry for your applications, Kyma's Telemetry module provides the following features:
+To support telemetry for your applications, the Telemetry module provides the following features:
 
 - Tooling for collection, filtering, and shipment: Based on the [Open Telemetry Collector](https://opentelemetry.io/docs/collector/) and [Fluent Bit](https://fluentbit.io/), you can configure basic pipelines to filter and ship telemetry data.
 - Integration in a vendor-neutral way to a vendor-specific observability system (traces and metrics only): Based on the [OpenTelemetry protocol (OTLP)](https://opentelemetry.io/docs/reference/specification/protocol/), you can integrate backend systems.
 - Guidance for the instrumentation (traces and metrics only): Based on [Open Telemetry](https://opentelemetry.io/), you get community samples on how to instrument your code using the [Open Telemetry SDKs](https://opentelemetry.io/docs/instrumentation/) in nearly every programming language.
-- [Enriching](gateways.md#data-enrichment) telemetry data by automatically adding common attributes (traces and metrics only). This is done in compliance with established semantic conventions, ensuring that the enriched data adheres to industry best practices and is more meaningful for analysis.
-- Opt-out from features for advanced scenarios: At any time, you can opt out for each data type, and use custom tooling to collect and ship the telemetry data.
-- SAP BTP as first-class integration: Integration into BTP Observability services is prioritized.
+- Enriching telemetry data by automatically adding common attributes. This is done in compliance with established semantic conventions, ensuring that the enriched data adheres to industry best practices and is more meaningful for analysis. For details, see [Data Enrichment](gateways.md#data-enrichment).
+- Opt-out of features for advanced scenarios: At any time, you can opt out for each data type, and use custom tooling to collect and ship the telemetry data.
+- SAP BTP as first-class integration: Integration into BTP Observability services, such as SAP Cloud Logging is prioritized. For more information, see [SAP Cloud Logging](integration/sap-cloud-logging/README.md). <!--- replace with Help Portal link once published? --->
 
 ## Scope
 
-The Telemetry module focuses only on the signals of application logs, distributed traces, and metrics. Other kinds of signals are not considered. Also, logs like audit logs or operational logs are not in scope.
+The Telemetry module focuses only on the signals of application logs, distributed traces, and metrics. Other kinds of signals are not considered. Also, audit logs are not in scope.  For audit logs, see [Auditing and Logging Information in Kyma](https://help.sap.com/docs/btp/sap-business-technology-platform/auditing-and-logging-information-in-kyma?locale=en-US&version=Cloud).
```

Comment:
The audit log setup is specific to the SAP product indeed, you will not have it in OSS. So either we remove the last sentence fully or have it only in DITA..

Reply:
removed with 301d0b8